### PR TITLE
[RFR][1LP] Nuage skel

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6652e56942a45032ddf8fd5258d54a8d235de9b9:

`cfme/networks/provider/__init__.py`:
- :bangbang: Line 115:44: **E261** *at least two spaces before inline comment*
- :bangbang: Line 125:44: **E261** *at least two spaces before inline comment*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/networks/provider/nuage.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                }
+
+
+class NuageEndpointForm(View):
+    @View.nested
+    class default(Tab, DefaultEndpointForm, BeforeFillMixin):  # NOQA
+        TAB_NAME = 'Default'
+        security_protocol = BootstrapSelect('default_security_protocol')
+        api_port = Input('default_api_port')
+
```
Comment (mshriver) [inline cfme/networks/provider/nuage.py]: **Question**
`hostname` in `NuageEndpoint.view_value_mapping`, but no widget here?

**Pull request merged**